client/SFTPClientGUI/SFTPClientGUI.py

```diff
@@ -0,0 +1,366 @@
+			current = self._tvmodel.iter_children(treeiter)
+		self._tvmodel.append(treeiter, [None, None, None, None, None, None, None])
+
+	def load_dirs(self, path, parent=None):
+		counter = 0
+		for name in self._yield_dir_list(path):
+			if self.local_hidden and name.startswith('.'):
+				continue
+			fullname = path + '/' + name
+			perm = self._check_perm(fullname)
+			raw_time = self._get_raw_time(fullname)
+			date_modified = '   ' + time.strftime('%y-%m-%d %H:%M:%S', time.localtime(raw_time))
+			is_folder = self._get_is_folder(fullname)
+			if is_folder:
+				icon = Gtk.IconTheme.get_default().load_icon('folder', 20, 0)
+				if perm[3] != 'r' and perm[5] != 'x':
+					icon = Gtk.IconTheme.get_default().load_icon('emblem-unreadable', 13, 0)
+				current = self._tvmodel.append(parent, (name, icon, fullname, perm, None, -1, date_modified))
+			else:
+				file_size = self._get_file_size(fullname)
+				hr_file_size = '   ' + boltons.strutils.bytes2human(file_size)
+				if perm[3] != 'r' and perm[5] != 'x':
+					icon = Gtk.IconTheme.get_default().load_icon('emblem-unreadable', 13, 0)
+				icon = Gtk.IconTheme.get_default().load_icon('empty', 12.5, 0)
+				current = self._tvmodel.append(parent, (name, icon, fullname, perm, hr_file_size, file_size, date_modified))
+			if is_folder and perm[3] == 'r' and perm[5] == 'x':
+				self._tvmodel.append(current, [None, None, None, None, None, None, None])
+			counter += 1
+		if counter < 1:
+			self._tvmodel.append(parent, [None, None, None, None, None, None, None])
+
+	def signal_menu_toggled_hidden_files(self, _):  # pylint: disable=method-hidden
+		self.local_hidden = not self.local_hidden
+		self.refresh()
+
+	def refresh(self):
+		model = self._tvmodel
+		exp_lines = []
+		model.foreach(lambda model, path, iter: exp_lines.append(path) if self.treeview_local.row_expanded(path) else 0)
+		self.treeview_local.collapse_all()
+		for path in exp_lines:
+			self.treeview_local.expand_row(path, False)
+
+	def signal_menu_activate_delete_prompt(self, _):
+		selection = self.treeview_local.get_selection()
+		model, treeiter = selection.get_selected()
+		dialog = Gtk.Dialog('Warning')
+		label = "Are you sure\n you want to delete this {0}?\n".format('directory' if model[treeiter][5] == -1 else 'file')
+		label = Gtk.Label(label)
+		label.set_justify(Gtk.Justification.CENTER)
+		dialog.vbox.pack_start(label, True, True, 0)
+		label.show()
+		dialog.add_button(Gtk.STOCK_CANCEL, Gtk.ResponseType.REJECT)
+		dialog.add_button(Gtk.STOCK_OK, Gtk.ResponseType.ACCEPT)
+		response_id = dialog.run()
+		dialog.destroy()
+		if response_id == Gtk.ResponseType.ACCEPT:
+			self.delete(model, treeiter)
+
+class LocalDirectory(DirectoryBase):
+	treeview_name = 'treeview_local'
+	def __init__(self, builder, logger, upload_button):
+		_treeview = builder.get_object('SFTPClientGUI.' + self.treeview_name)
+		super(LocalDirectory, self).__init__(_treeview, logger)
+		self.upload_button = upload_button
+		init_dir = os.getcwd()
```

Comment:
This should be the root of the drive since users can't move up directories right now.